tests/test_token_warning_fix.py

```
@@ -0,0 +1,142 @@
+
+
+class TestTokenWarningFix:
+    """Test token warning consistency fixes."""
+
+    @patch("fetchtastic.downloader._initial_setup_and_config")
+    @patch("fetchtastic.downloader._check_wifi_connection")
+    @patch("fetchtastic.downloader._process_firmware_downloads")
+    @patch("fetchtastic.downloader._process_apk_downloads")
+    @patch("fetchtastic.downloader._finalize_and_notify")
+    def test_main_shows_token_warning_consistently(
+        self, mock_finalize, mock_apk, mock_firmware, mock_wifi, mock_setup
```

Ruff (0.14.2)

Unused method argument: `mock_finalize`

(ARG002)